apicapi/apic_client.py

```diff
@@ -278,11 +282,15 @@ def name(self, *params):
 
 
 class ApicSession(object):
+    HTTP_GET = 'GET'
+    HTTP_POST = 'POST'
+    HTTP_PUT = 'PUT'
+    HTTP_DELETE = 'DELETE'
```

Comment:
Prefer to keep the prefix, symbolic names like GET, PUT etc are not very informative